File: python/tests/integration/copy_.py
import sys
from pathlib import Path

ROOT = Path(__file__).resolve().parents[3]
if str(ROOT) not in sys.path:
    sys.path.insert(0, str(ROOT))

from python.lira.arch_ser_yaml import copy_arch


if __name__ == '__main__':
    if len(sys.argv) != 3:
        sys.exit('Usage: copy_.py <INPUT> <OUTPUT>')
    copy_arch(Path(sys.argv[1]), Path(sys.argv[2]))

File: ruby/tests/integration/copy.rb
$LOAD_PATH.unshift(File.expand_path('../..', __dir__))
require 'lira'
include Lira

if ARGV.length != 2
  $stderr.puts "Usage: #{$0} <input.yaml> <output.yaml>"
  exit 1
end

ArchSerYaml.copy_arch(ARGV[0], ARGV[1])
